impl/src/main/java/io/kroxylicious/testing/kafka/invm/InVMKafkaCluster.java

```diff
@@ -182,7 +184,11 @@ public void start() {
             }
         }
 
-        servers.stream().parallel().forEach(Server::startup);
+        servers.stream().parallel().forEach(server -> Awaitility.await().atMost(Duration.ofSeconds(5)).pollDelay(Duration.ofMillis(50)).until(() -> {
```

Comment:
Wondering if 5 seconds is a bit tight for slow machines

Reply:
I was hesitant to go longer as its all happening in cluster setup so could be quite slow for extension users. However a working cluster is probably more important than speed so will bump it up.